compiler/quilt/tools/store.py

```diff
@@ -376,7 +376,16 @@ def _read_parquet_arrow(self, hash_list):
         objfiles = [self.object_path(h) for h in hash_list]
         dataset = ParquetDataset(objfiles)
         table = dataset.read(nthreads=4)
-        dataframe = table.to_pandas()
+        try:
+            dataframe = table.to_pandas()
+        except:
```

Comment:
Catch a more specific exception - `AssertionError`, `Exception`, etc..; you don't want to catch things like `KeyboardInterrupt` or `SystemExit`.